src/elements/cta/src/stories.tsx

```diff
@@ -26,6 +26,7 @@ export const Example = () => {
     "Find the best credit card for you, whether you're looking for a 0% card for balance transfers or purchases or day to day spending and rewards."
   )
   const buttonText = text('Button text', 'Compare credit cards')
+  const buttonProps = object('Button Props', {})
```

Comment:
an example of button probs being used to add an onclick would be nice

Reply:
I've added an example to set a target on the anchor tag.

In general, I suspect people won't be using this to add onclick events - they should be adding `data-nerd-...` attributes for tracking, rather than rolling JS functions. Also, it looks like story book doesn't play nicely with functions in json props.